ibm/service/scc/resource_ibm_scc_posture_import_profile.go

```diff
@@ -0,0 +1,196 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package scc
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/scc-go-sdk/v4/posturemanagementv2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceIBMSccPostureProfileImport() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMSccPostureProfileImport,
+		ReadContext:   resourceIBMSccPostureProfileImportRead,
+		UpdateContext: resourceIBMSccPostureProfileImportRead,
+		DeleteContext: resourceIBMSccPostureProfileImportDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"file": {
+				Type:        schema.TypeString,
+				Description: "File to import",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"profile_id": {
```

Comment:
@hkantare profile_id is not needed, I removed it. 
But when using ForceNew for a field, we need to have atleast one optional field. That's the reason I make name as optional: true along with computed.

When I remove optional: true I am getting below error. But As per the import file feature we need ForceNew field set to be true, otherwise when we upload new file with another name the tfstate is not updating.

"All fields are ForceNew or Computed w/out Optional, Update is superfluous".

So, That's reason name field make it optional true along with computed.